src/stirling/source_connectors/tcp_stats/tcp_stats_bpf_test.cc

```diff
@@ -46,26 +46,19 @@ class TcpTraceTest : public TcpTraceBPFTestFixture {};
 //-----------------------------------------------------------------------------
 
 TEST_F(TcpTraceTest, Capture) {
+  std::thread server_thread([&] { ASSERT_OK(px::Exec("nc -l -p 12345 -s 127.0.0.1 -v")); });
+  server_thread.detach();
+
   StartTransferDataThread();
 
   // Send "hello" to 127.0.0.1, total 6 bytes of data
-  std::string cmd1 = "echo \"hello\" | nc 127.0.0.1 22";
+  std::string cmd1 = "echo \"hello\" | nc -w1 127.0.0.1 12345 -v";
   ASSERT_OK(px::Exec(cmd1));
 
   StopTransferDataThread();
   auto records = testing::ExtractToString(tcp_stats::kTCPStatsTable, tcp_stats_table_);
 
-  EXPECT_THAT(records, HasSubstr("remote_addr:[127.0.0.1] remote_port:[22] tx:[6]"));
-
-  StartTransferDataThread();
-
-  // Send "This is sample test" to 127.0.0.1, total 20 bytes of data
-  std::string cmd2 = "echo \"This is sample test\" | nc 127.0.0.1 22";
-  ASSERT_OK(px::Exec(cmd2));
-
-  StopTransferDataThread();
-  auto records2 = testing::ExtractToString(tcp_stats::kTCPStatsTable, tcp_stats_table_);
-  EXPECT_THAT(records2, HasSubstr("remote_addr:[127.0.0.1] remote_port:[22] tx:[20]"));
```

Comment:
I decided to remove this assertion since it didn't provide any more coverage than the one earlier in the test.